src/bitbucket/BitbucketAPI.ts

```diff
@@ -51,12 +64,25 @@ export class BitbucketAPI {
         attemptNumber,
         attemptsLeft,
         pullRequestId,
+        requestId,
+        targetBranch,
       });
     };
     await pRetry(attemptMerge, { onFailedAttempt, retries: 5 })
-      .then(() => Logger.info('Merged Pull Request', { pullRequestId }))
+      .then(() =>
+        Logger.info('Merged Pull Request', {
+          namespace: 'bitbucket:api:mergePullRequest',
+          requestId,
+          pullRequestId,
+        }),
+      )
       .catch(err => {
-        Logger.error('Unable to merge pull request', { err, pullRequestId });
+        Logger.error('Unable to merge pull request', {
+          namespace: 'bitbucket:api:mergePullRequest',
+          err,
```

Comment:
I recall some sort of issue before with loggin err objects... 
Do you remember what that was? It was something super weird....

Reply:
Im pretty sure this is fine, but trying to recall